.azure-pipelines/build-template.yml

```diff
@@ -50,25 +61,33 @@ jobs:
       sudo apt-get install -y ss-dev libldap2-dev libc6-dev libkeyutils-dev byacc docbook-to-man libsasl2-dev libverto-dev python3-cheetah python3-lxml doxygen doxygen-latex tex-gyre
 
       sudo pip3 install blurb
+
       mkdir -p $(Pipeline.Workspace)/target
     displayName: 'Install packages'
   - checkout: self
     submodules: true
   - script: |
       cd src/SymCrypt
-      git submodule update --init -- jitterentropy-library
+      git submodule update --init -- 3rdparty/jitterentropy-library
     displayName: 'Checkout Symcrypt submodules'
   - script: |
       set -ex
       sudo mkdir -p $HOME
       sudo pip3 install -r src/SymCrypt/scripts/requirements.txt
-      ARCH=${{ parameters.arch }} make symcrypt
-      sudo dpkg -i target/symcrypt-openssl*.deb
+      git config --global user.name sonicbld
+      git config --global user.email sonicbld@microsoft.com
+      if [ "${{ parameters.arch }}" == armhf ]; then
+        ARCH=${{ parameters.arch }} SYMCRYPT_TEST=n make symcrypt
+      else
+        ARCH=${{ parameters.arch }} make symcrypt
+        sudo dpkg -i target/symcrypt-openssl*.deb
```

Comment:
Not entirely sure what this line is doing, but it looks like you're doing it twice now for non-armhf?

Reply:
For armhf, it is built in the arm64 linux32 env to improve the performance, but the test is disabled.
The artifacts will be copied to the amd64 env, and run the tests in another test azp job.

Reply:
Are both OpenSSL and SCOSSL built in linux32?
I just don't see why you need to do the `sudo dpkg -i target/symcrypt-openssl*.deb` twice?

Reply:
Yes, OpenSSL and SCOSSL buld in linux32, the amd64 qemu is only for test.
Removed the duplicate installation for symcrypt-openssl (removed line 90).